openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -0,0 +1,649 @@
+    :class:`openfe.gromacs.protocols.gromacs_md.GromacsMDProtocolSettings`
+    :class:`openfe.gromacs.protocols.gromacs_md.GromacsMDProtocolUnit`
+    :class:`openfe.gromacs.protocols.gromacs_md.GromacsMDProtocolResult`
+    """
+
+    result_cls = GromacsMDProtocolResult
+    _settings: GromacsMDProtocolSettings
+
+    @classmethod
+    def _default_settings(cls):
+        """A dictionary of initial settings for this creating this Protocol
+
+        These settings are intended as a suitable starting point for creating
+        an instance of this protocol.  It is recommended, however that care is
+        taken to inspect and customize these before performing a Protocol.
+
+        Returns
+        -------
+        Settings
+          a set of default settings
+        """
+        return GromacsMDProtocolSettings(
+            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(
+                nonbonded_method="PME",
+                constraints=None,
+            ),
+            thermo_settings=settings.ThermoSettings(
+                temperature=298.15 * unit.kelvin,
+                pressure=1 * unit.bar,
+            ),
+            partial_charge_settings=OpenFFPartialChargeSettings(),
+            solvation_settings=OpenMMSolvationSettings(),
+            engine_settings=OpenMMEngineSettings(),
+            integrator_settings=IntegratorSettings(),
+            simulation_settings_em=EMSimulationSettings(
+                integrator="steep",
+                nsteps=5000,
+                tcoupl="no",
+                pcoupl="no",
+                gen_vel="no",
+            ),
+            simulation_settings_nvt=NVTSimulationSettings(
+                nsteps=50000,  # 100ps
+                pcoupl="no",
+                gen_vel="yes",
+            ),
+            simulation_settings_npt=NPTSimulationSettings(
+                nsteps=500000,  # 1ns
+                pcoupl="Parrinello-Rahman",
+                gen_vel="no",  # If continuation from NVT simulation
+            ),
+            output_settings_em=EMOutputSettings(
+                mdp_file="em.mdp",
+            ),
+            output_settings_nvt=NVTOutputSettings(
+                mdp_file="nvt.mdp",
+            ),
+            output_settings_npt=NPTOutputSettings(
+                mdp_file="npt.mdp",
+                nstxout=5000,
+                nstvout=5000,
+                nstfout=5000,
+                nstxout_compressed=5000,
+            ),
+            protocol_repeats=1,
+            gro="system.gro",
+            top="system.top",
+        )
+
+    def _create(
+        self,
+        stateA: ChemicalSystem,
+        stateB: ChemicalSystem,
+        mapping: dict[str, gufe.ComponentMapping] | None = None,
+        extends: gufe.ProtocolDAGResult | None = None,
+    ) -> list[gufe.ProtocolUnit]:
+
+        # Validate solvent component
+        # TODO: Maybe only allow PME as nonbonded_method since Interchange requires PME?
```

Comment:
What do you mean by interchange requires PME?

Reply:
Interchange throws an error if e.g. `no_cutoff` instead of `pme` is used.

Reply:
Removed this as we have a check for making sure that a `SolventComponent` has to be provided and there is an error anyways if there is a `SolventComponent` and `nocutoff`.